tools/OpenConsole.psm1

```diff
@@ -1,3 +1,10 @@
+# throw error if Powershell version is lower than 7
+if ($PSVersionTable.PSVersion.Major -lt 7) {
+    throw "This script requires PowerShell 7 or higher. Current version: $($PSVersionTable.PSVersion)"
+    exit
+} 

```

Comment:
Why not use `#Requires -Version 7`? https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_requires

Reply:
Thank you, I changed it to using #Requires.